### PR TITLE
murmur: exit(1) when encountering a qFatal() when using syslog logging.

### PR DESCRIPTION
**Pull request by mkrautz**

### CODE REVIEW

**Comment by mkrautz** [comment]
@hacst PTAL.

File: src/murmur/main.cpp [hunk 1]
```diff
@@ -84,6 +84,9 @@ static void murmurMessageOutputQString(QtMsgType type, const QString &msg) {
 			break;
 		}
 		syslog(level, "%s", qPrintable(msg));
+		if (type == qtFatalMsg) {
```
Comment (hacst) [inline src/murmur/main.cpp]: Should be ```==QtFatalMsg```

**Comment by mkrautz** [comment]
@hacst 

PTAL

**Comment by hacst** [comment]
LGTM

**Comment by mkrautz** [comment]
Landed as abdb5004bb92b28271673e33df9cc3e314f5711a.

**Pull request closed without merging**